hcm/bish

```diff
@@ -11,7 +11,7 @@ from hcm import types
 SAMPLE_RATE = 8000  # hz
 INTERVAL_LENGTH = 1000  # ms
 
```

Comment:
I see you moved the file from `bin`. Please update `setup.py` to point to this new location of bish: 
```
scripts=['hcm/bish']
```

Reply:
I'm sure there was a better way than moving bish into the main directory, but for whatever reason it wasn't finding hcm when I tried running it out of the bin folder.